pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -77,8 +81,93 @@ func (pl *CSILimits) Name() string {
 func (pl *CSILimits) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
 		{Event: framework.ClusterEvent{Resource: framework.CSINode, ActionType: framework.Add}},
-		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}},
+		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.Delete}, QueueingHintFn: pl.isSchedulableAfterPodDeleted},
+	}
+}
+
+func (pl *CSILimits) isSchedulableAfterPodDeleted(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	deletedPod, _, err := util.As[*v1.Pod](oldObj, newObj)
+	if err != nil {
+		logger.V(5).Error(err, "Unexpected objects in isSchedulableAfterPodDeleted", "oldObj", oldObj)
```

Comment:
remove all error logs right before returning error. The scheduling queue logs errors.
https://github.com/kubernetes/kubernetes/blob/76cd7521aa6216fb069dbd2d1f0f2c7a37fba32f/pkg/scheduler/internal/queue/scheduling_queue.go#L455-L465